### PR TITLE
Login

### PR DESCRIPTION
**Pull request by SimonLeger**

### CODE REVIEW

File: backend/swagger-output.json [hunk 1]
```diff
@@ -831,7 +888,7 @@
       "post": {
         "tags": ["Réservations"],
         "summary": "Créer une réservation",
-        "description": "Nécessite d'être de niveau 2",
```
Comment (levydanqc) [inline backend/swagger-output.json:834]: Why did you delete this line ? If you didn't wanted to change this file you can remove it from this PR

File: frontend/src/views/LoginView.vue [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      />
+      <div style="width: 100%"></div>
+      <input
+        class="text-center p-1 rounded"
+        style="background-color: #e3e3e3"
+        v-model="motdepasse"
+        placeholder="Mot de passe"
+        type="password"
+      />
+      <div style="width: 100%"></div>
```
Comment (levydanqc) [inline frontend/src/views/LoginView.vue]: Pretty much the same as signup around here, juste make sure not to use the 'style' attribute inside html and use bootstrap classes like you're currently doing !

**Review by levydanqc — CHANGES_REQUESTED** [review]

File: yarn-error.log [hunk 3]
```diff
@@ -0,0 +1,503 @@
+Arguments:
```
Comment (levydanqc) [inline yarn-error.log]: This file shouldn't be here.

**Review by levydanqc — CHANGES_REQUESTED** [review]

File: frontend/src/views/LoginView.vue [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+      })
+        .then((response) => {
+          if (response.status === 200) {
+            return response.json();
+          } else {
+            this.erreurs.push("Erreur !");
+          }
+        })
+        .then((data) => {
+          console.log("data", data);
```
Comment (levydanqc) [inline frontend/src/views/LoginView.vue]: You should remove console statement as mentioned in the Github Action Warning juste below

**Review by levydanqc — CHANGES_REQUESTED** [review]

**Comment by SimonLeger** [comment]
I corrected everything, let me know if anything needs more attention

**Review by levydanqc — DISMISSED** [review]
Perfect! I'll let you merge it (and don't forget to squash) 🥇

**Review by levydanqc — APPROVED** [review]

**Pull request merged**